src/odemis/driver/test/xt_client_test.py

```diff
@@ -276,6 +277,66 @@ def test_set_beam_shift(self):
         self.scanner.beamShift.value = (new_beam_shift_x, new_beam_shift_y)
         self.assertAlmostEqual((new_beam_shift_x, new_beam_shift_y), self.scanner.beamShift.value)
 
+    def test_pitch(self):
+        """Test setting and getting the pitch between two beams within the multiprobe pattern."""
+        pitch_range = self.microscope.pitch_info()["range"]
+        current_pitch = self.microscope.get_pitch()
+        new_pitch = current_pitch + 1
+        new_pitch = new_pitch if pitch_range[0] < new_pitch < pitch_range[1] else current_pitch - 1
+        self.microscope.set_pitch(new_pitch)
+        self.assertAlmostEqual(new_pitch, self.microscope.get_pitch())
+        self.microscope.set_pitch(current_pitch)
+
+    def test_primary_stigmator(self):
+        """Test getting and setting the primary stigmator."""
+        if self.xt_type != 'xttoolkit':
+            self.skipTest("This test needs XTToolkit to run.")
+        stig_range = self.microscope.primary_stigmator_info()["range"]
+        init_stig = self.microscope.get_primary_stigmator()
+        stig_x, stig_y = (init_stig[0] + 1e-3, init_stig[1] + 1e-3)
+        stig_x = stig_x if stig_range['x'][0] < stig_x < stig_range['x'][1] else init_stig[0] - 1e-3
+        stig_y = stig_y if stig_range['y'][0] < stig_y < stig_range['y'][1] else init_stig[1] - 1e-3
+        new_stig = (stig_x, stig_y)
+        self.microscope.set_primary_stigmator(*new_stig)
+        self.assertEqual(self.microscope.get_primary_stigmator()[0], new_stig[0])
+        self.assertEqual(self.microscope.get_primary_stigmator()[1], new_stig[1])
+        self.microscope.set_primary_stigmator(*init_stig)
```

Comment:
Maybe mention that this is important to set back the old values so the system is not suddenly misaligned. Same for secondary stigmator.